Source/EasyNetQ.sln

```diff
@@ -59,10 +59,10 @@ Project("{8BB2217D-0F2D-49D1-97BC-3654ED321F3B}") = "EasyNetQ.DI.Windsor", "Easy
 EndProject
```

Comment:
This can be undone because the projects only changed position within the file and nothing is added here.